spyder_notebook/notebookplugin.py

```diff
@@ -74,14 +75,19 @@ def __init__(self, parent, testing=False):
                                              icon=ima.icon('project_expanded'),
                                              tip=_('Open a new notebook'),
                                              triggered=self.create_new_client)
+        open_console_btn = create_toolbutton(self,
+                                             icon=ima.icon('ipython_console'),
+                                             tip=_('Open the IPython console'),
+                                             triggered=self.open_console)
         menu_btn = create_toolbutton(self, icon=ima.icon('tooloptions'),
                                      tip=_('Options'))
 
         self.menu = QMenu(self)
         menu_btn.setMenu(self.menu)
         menu_btn.setPopupMode(menu_btn.InstantPopup)
         add_actions(self.menu, self.menu_actions)
-        corner_widgets = {Qt.TopRightCorner: [new_notebook_btn, menu_btn]}
+        corner_widgets = {Qt.TopRightCorner: [new_notebook_btn,
+                                              open_console_btn, menu_btn]}

```

Comment:
Let's remove `open_console_btn` from here. It takes too much space on the tabbar.